tests/sspi_test.py

```diff
@@ -44,11 +44,9 @@ def test_sec_context(self):
 
         token_buf = create_string_buffer(10000)
         bufs = [(pytds.sspi.SECBUFFER_TOKEN, token_buf)]
-        server = settings.HOST
-        if "\\" in server:
-            server, _ = server.split("\\")
-        host, _, _ = socket.gethostbyname_ex(server)
-        target_name = "MSSQLSvc/{0}:1433".format(host)
+        # getting current host's name
+        host, _, _ = socket.gethostbyname_ex("")
```

Comment:
IDK about other versions of python, but at least on 3.9 I get a `Resolver Error` when a blank str is given to `socket.gethostbyname_ex`.

Reply:
Actually, it doesn't seem to work with 3.11, either.

Reply:
Thanks for reviewing, will take a look.
Weirdly tests passed on Python 3.8 https://ci.appveyor.com/project/denisenkom/pytds/builds/50784050

Reply:
This works on Windows on Python 3.8 and 3.12 and this test is only executed on Windows.